src/main/java/uk/gov/register/views/ViewFactory.java

```diff
@@ -87,66 +92,106 @@ public HomePageView homePageView(int totalRecords, int totalEntries, Optional<In
         return getAttributionView("entry.html", entry);
     }
 
-    public PaginatedView<EntryListView> getEntriesView(Collection<Entry> entries, Pagination pagination) {
+    public PaginatedView<EntryListView> getEntriesView(final Collection<Entry> entries, final Pagination pagination) {
         return new PaginatedView<>("entries.html", requestContext, getRegistry(), getBranding(), register.get(), registerTrackingConfiguration.get(), registerResolver, pagination, new EntryListView(entries));
     }
 
-    public PaginatedView<EntryListView> getRecordEntriesView(String recordKey, Collection<Entry> entries, Pagination pagination) {
+    public EntryListView getEntriesView(final Collection<Entry> entries) {
+        return new EntryListView(entries);
+    }
+
+    public PaginatedView<EntryListView> getRecordEntriesView(final String recordKey, final Collection<Entry> entries, final Pagination pagination) {
         return new PaginatedView<>("entries.html", requestContext, getRegistry(), getBranding(), register.get(), registerTrackingConfiguration.get(), registerResolver, pagination, new EntryListView(entries, recordKey));
     }
 
-    public AttributionView<RecordView> getRecordView(RecordView record) {
+    public AttributionView<RecordView> getRecordView(final RecordView record) {
         return getAttributionView("record.html", record);
     }
 
-    public PaginatedView<RecordsView> getRecordsView(Pagination pagination, RecordsView recordsView) {
+    public PaginatedView<RecordsView> getRecordsView(final Pagination pagination, final RecordsView recordsView) {
         return new PaginatedView<>("records.html", requestContext, getRegistry(), getBranding(), register.get(), registerTrackingConfiguration.get(), registerResolver, pagination,
                 recordsView);
     }
 
-    public ItemView getItemMediaView(Item item) {
+    public ItemView getItemMediaView(final Item item) {
         return new ItemView(item.getSha256hex(), itemConverter.convertItem(item, register.get().getFieldsByName()), getFields());
     }
 
-    public RecordView getRecordMediaView(Record record) {
+    public RecordView getRecordMediaView(final Record record) {
         return new RecordView(record, register.get().getFieldsByName(), itemConverter);
     }
 
-    public RecordsView getRecordsMediaView(List<Record> records) {
+    public RecordsView getRecordsMediaView(final List<Record> records) {
         return new RecordsView(records, register.get().getFieldsByName(), itemConverter, false, false);
     }
 
-    public RecordsView getIndexRecordsMediaView(List<Record> records) {
+    public RecordsView getIndexRecordsMediaView(final List<Record> records) {
         return new RecordsView(records, register.get().getFieldsByName(), itemConverter, false, true);
     }
 
+    public PreviewRecordPageView previewRecordsPageView(final List<Record> records, final String key, final String previewType) {
```

Comment:
Let's follow the convention here and have a `get` prefix

Reply:
You can find both cases in the file. Ex: `downloadPageView`

Reply:
Discussed.